File: AtCoder/AtCoder11.25/B.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solve(vector<int> a, int l, int r, int n){
    vector<int> resultado;
    int xi;
    for(auto ai : a){
        if(ai < l) xi = l;
        else if (ai > r) xi = r;
        else xi = ai;
        resultado.push_back(xi);
    }
    return resultado;
}

int main(){
    int n, l, r; cin >> n >> l >> r;
    vector<int> a;
    for(int i = 0; i < n; i++){
        int temp; cin >> temp;
        a.push_back(temp);
    }
    vector<int> ans = solve(a, l, r, n);
    for(unsigned int i = 0; i < ans.size(); i++){
        cout << ans[i] << " ";
    }
    cout << endl;
}

File: AtCoder/AtCoder11.25/B.py
def solve(A, L, R):
    ans = []

    for Ai in A:
        if Ai < L:
            Xi = L
        elif Ai > R:
            Xi = R
        else:
            Xi = Ai
        ans.append(Xi)

    return ans

n, l, r = map(int,input().split())
a = list(map(int,input().split()))
ans = solve(a, l, r)
print(*ans)


